pkgs/top-level/all-packages.nix

```diff
@@ -29109,8 +29109,6 @@ with pkgs;
 
   taxi = callPackage ../applications/networking/ftp/taxi { };
 
-  taxi-cli = with python3Packages; toPythonApplication taxi;
```

Comment:
Why are you removing this if it is a cli and library?

Reply:
I noticed the cli part is already bundled directly in the repo thanks to a flake. I feel like having the cli also in the nixpkgs would duplicate this effort and increase the maintenance complexity of this package.

Reply:
Projects that have flakes are nice but it is not that easy to track only releases with flakes. Also having a flake does not exclude a project from inclusion in nixpkgs.

> increase the maintenance complexity of this package.

I don't think so that will happen. Also requiring people to use an extra flake when we already have the package is very inconvenient.

Reply:
Restored package